lib/rust/lexer/definition/src/lexer.rs

```diff
@@ -81,6 +88,10 @@ impl EnsoLexer {
     pub fn append_token(&mut self, token:Token) {
         debug!(self.logger,"Append: {&token:?}");
         self.output.append(token);
+        if self.block_state.has_delayed_lines() {
+            let tokens = self.consume_tokens();
+            self.block_state.append_line_to_current(tokens.into());
```

Comment:
what are `delayed_lines` used for? What is an example use case they are needed for?